web/src/components/RawDataEditor/RawDataEditor.tsx

```diff
@@ -154,19 +154,19 @@ const RawDataEditor: React.FC<Props> = ({ visible, readonly = true, type, data =
                 handleModeChange(value);
               }}
               data-cy='code-mirror-mode'
-            ></Select>,
+            />,
```

Comment:
> You can use the self-closing form of the `Select tag` since there are no children in it

I don't understand, can you explain more or provide suggestion?

Reply:
ops, it is already a self-closing tag, forget this review ^_^